src/ethereum/base_types.py

```diff
@@ -1,18 +1,23 @@
 """
-Numeric & Array Types
-^^^^^^^^^^^^^^^^^^^^^
-
-.. contents:: Table of Contents
-    :backlinks: none
-    :local:
-
-Introduction
-------------
-
 Integer and array types which are used by—but not unique to—Ethereum.
-"""
 
-from __future__ import annotations
+[`Uint`] represents non-negative integers of arbitrary size, while subclasses
+of [`FixedUint`] (like [`U256`] or [`U32`]) represent non-negative integers of
+particular sizes.
+
+Similarly, [`Bytes`] represents arbitrarily long byte sequences, while
+subclasses of [`FixedBytes`] (like [`Bytes0`] or [`Bytes64`]) represent
+sequences containing an exact number of bytes.
+
+[`Uint`]: ref:ethereum.base_types.Uint
+[`FixedUint`]: ref:ethereum.base_types.FixedUint
+[`U32`]: ref:ethereum.base_types.U32
+[`U256`]: ref:ethereum.base_types.U256
+[`Bytes`]: ref:ethereum.base_types.Bytes
+[`FixedBytes`]: ref:ethereum.base_types.FixedBytes
+[`Bytes0`]: ref:ethereum.base_types.Bytes0
+[`Bytes64`]: ref:ethereum.base_types.Bytes64
```

Comment:
```suggestion
[`Uint`]: ref:ethereum.base_types.Uint
[`FixedUint`]: ref:ethereum.base_types.FixedUint
[`U32`]: ref:ethereum.base_types.U32
[`U64`]: ref:ethereum.base_types.U64
[`U256`]: ref:ethereum.base_types.U256
[`Bytes`]: ref:ethereum.base_types.Bytes
[`FixedBytes`]: ref:ethereum.base_types.FixedBytes
[`Bytes0`]: ref:ethereum.base_types.Bytes0
[`Bytes4`]: ref:ethereum.base_types.Bytes4
[`Bytes8`]: ref:ethereum.base_types.Bytes8
[`Bytes20`]: ref:ethereum.base_types.Bytes20
[`Bytes32`]: ref:ethereum.base_types.Bytes32
[`Bytes64`]: ref:ethereum.base_types.Bytes64
[`Bytes256`]: ref:ethereum.base_types.Bytes256
```

Reply:
These are markdown [reference links](https://spec.commonmark.org/0.31.2/#reference-link), and are only necessary if a link exists in the text.

Unless you're suggesting that I also add all the types to the text?

Reply:
Ah. Got it